addons/mail/static/src/models/messaging_initializer.js

```diff
@@ -225,44 +219,10 @@ registerModel({
             }));
         },
         /**
-         * @param {object} resUsersSettings
-         * @param {integer} resUsersSettings.id
-         * @param {boolean} resUsersSettings.is_discuss_sidebar_category_channel_open
-         * @param {boolean} resUsersSettings.is_discuss_sidebar_category_chat_open
-         * @param {boolean} resUsersSettings.use_push_to_talk
-         * @param {String} resUsersSettings.push_to_talk_key
-         * @param {number} resUsersSettings.voice_active_duration
-         * @param {Object} [resUsersSettings.volume_settings]
+         * @param {Object} resUsersSettingsData
          */
-        _initResUsersSettings({
-            id,
-            is_discuss_sidebar_category_channel_open,
-            is_discuss_sidebar_category_chat_open,
-            use_push_to_talk,
-            push_to_talk_key,
-            voice_active_duration,
-            volume_settings = [],
-        }) {
-            this.messaging.currentUser.update({ resUsersSettingsId: id });
-            this.messaging.update({
-                userSetting: insertAndReplace({
-                    id,
-                    usePushToTalk: use_push_to_talk,
-                    pushToTalkKey: push_to_talk_key,
-                    voiceActiveDuration: voice_active_duration,
-                    volumeSettings: volume_settings,
-                }),
-            });
-            this.messaging.discuss.update({
-                categoryChannel: insertAndReplace({
-                    isServerOpen: is_discuss_sidebar_category_channel_open,
-                    serverStateKey: 'is_discuss_sidebar_category_channel_open',
-                }),
-                categoryChat: insertAndReplace({
-                    isServerOpen: is_discuss_sidebar_category_chat_open,
-                    serverStateKey: 'is_discuss_sidebar_category_chat_open',
-                }),
-            });
+        _initResUsersSettings(resUsersSettingsData) {
+            this.messaging.currentUser.update({ resUsersSettings: insertAndReplace(resUsersSettingsData) });
```

Comment:
ideally should be insert global, link to user should be done through user_id field of setting